src/dashboard/tests/test_helpers.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import __builtin__
+import json
+import os
+import requests
+import tempfile
```

Comment:
You don't need to import a module from the standard library in order to mock part of it's functionality. You can mock it without import it like this:

```
mocker.patch("os.path.isfile", return_value=False)
```